src/main/java/org/example/bookstore/model/status/Status.java

```diff
@@ -0,0 +1,7 @@
+package org.example.bookstore.model.status;
+
+public enum Status {
```

Comment:
```suggestion
public enum OrderStatus {
```
Or make it a nested class of `Order`, up to you